### PR TITLE
feat(policies): Data Source for managed policies

### PR DESCRIPTION
**Pull request by ben-m-lucas**
This PR adds a new data source `sysdig_secure_managed_policy` that can be used for getting data from managed policies.

### CODE REVIEW

File: website/docs/r/secure_managed_policy.md [hunk 1]
```diff
@@ -46,7 +46,8 @@ resource "sysdig_secure_managed_policy" "sysdig_runtime_threat_detection" {
 
 * `name` - (Required) The name of the Secure managed policy. It must match the name of an existing managed policy.
 
-* `type` - (Optional) Specifies the type of the runtime policy. Must be one of: `falco`, `list_matching`, `k8s_audit`, `aws_cloudtrail`. By default it is `falco`.
+* `type` - (Optional) Specifies the type of the runtime policy. Must be one of: `falco`, `list_matching`, `k8s_audit`,
```
Comment (kmvachhani) [inline website/docs/r/secure_managed_policy.md:49]: do we have gcp/azure as supported policy types?

Comment (ben-m-lucas) [reply website/docs/r/secure_managed_policy.md:49]: Yes, we do.  I checked in the API and the validation that terraform performs also allows gcp/azure.  We just never updated the documentation.

**Review by kmvachhani — APPROVED** [review]
LGTM!

File: website/docs/d/secure_managed_policy.md
Comment (ombellare) [inline website/docs/d/secure_managed_policy.md:1]: Out of curiosity, any reason for the docs folder being named `d` and `r`?

**Review by ombellare — APPROVED** [review]

File: website/docs/d/secure_managed_policy.md
Comment (ben-m-lucas) [reply website/docs/d/secure_managed_policy.md:1]: `r` is for the resources and `d` is for the data sources.  Not sure why the abbreviated version was chosen.

Comment (ben-m-lucas) [reply website/docs/d/secure_managed_policy.md:1]: ah, other than it seems like other providers structure themselves that way as well:
https://github.com/hashicorp/terraform-provider-azurerm/tree/main/website/docs
https://github.com/hashicorp/terraform-provider-aws/tree/main/website/docs

**Pull request merged**